pkgs/by-name/pr/proton-ge-bin/package.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+, writeScript
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "proton-ge-bin";
+  version = "GE-Proton9-1";
+
+  src = fetchzip {
+    url = "https://github.com/GloriousEggroll/proton-ge-custom/releases/download/${finalAttrs.version}/${finalAttrs.version}.tar.gz";
+    hash = "sha256-odpzRlzW7MJGRcorRNo784Rh97ssViO70/1azHRggf0=";
+  };
+
+  outputs = [ "out" "steamcompattool" ];
+
+  buildCommand = ''
+    runHook preBuild
+
+    # Provide an error message when adding it to an env
+    echo "${finalAttrs.pname} should not be installed into environments. Please use programs.steam.extraCompatPackages." > $out
+
+    ln -s $src $steamcompattool
+
+    runHook postBuild
+  '';
+
+  /*
+    We use the created releases, and not the tags, for the update script as nix-update loads releases.atom
+    that contains both. Sometimes upstream pushes the tags but the Github releases don't get created due to
+    CI errors. Last time this happened was on 8-32, where a tag was created but no releases were created.
```

```suggestion
    CI errors. Last time this happened was on 8-33, where a tag was created but no releases were created.
```

actually it was 8-33 https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton8-33  which has no release https://github.com/GloriousEggroll/proton-ge-custom/releases 
i did state that out wrong on the first comment, but when noting it down here, we should be correct with the numbers i guess

---

This commit is from @NotAShelf's PR; GH just shows it here because it's not in master.

---

@NotAShelf would you prefer to adjust this in your PR and have it merged on its own or do we want to continue the proton-ge init here?

---

lets continue it here